### ISSUE TITLE
Possible integration point with openai/whisper output 

### ISSUE THREAD

**Issue by ArabCoders** [tkarabela/pysubs2#58]
Hello, I was wondering if it possible to add a new method to support [whisper transcribe](https://github.com/openai/whisper) output and convert that to the supported output formats.

This is currently how it's done at whisper

https://github.com/openai/whisper/blob/9f70a352f9f8630ab3aa0d06af5cb9532bd8c21d/whisper/utils.py#L63
```python
def write_srt(transcript: Iterator[dict], file: TextIO):
    """
    Write a transcript to a file in SRT format.
    Example usage:
        from pathlib import Path
        from whisper.utils import write_srt
        result = transcribe(model, audio_path, temperature=temperature, **args)
        # save SRT
        audio_basename = Path(audio_path).stem
        with open(Path(output_dir) / (audio_basename + ".srt"), "w", encoding="utf-8") as srt:
            write_srt(result["segments"], file=srt)
    """
    for i, segment in enumerate(transcript, start=1):
        # write srt lines
        print(
            f"{i}\n"
            f"{format_timestamp(segment['start'], always_include_hours=True, decimal_marker=',')} --> "
            f"{format_timestamp(segment['end'], always_include_hours=True, decimal_marker=',')}\n"
            f"{segment['text'].strip().replace('-->', '->')}\n",
            file=file,
            flush=True,
        )
```

**Comment by tkarabela** [comment]
Hi, I'm happy to add support for this :) As I understand it, you would like to be able to do something like the following?

```py
import whisper
import pysubs2

model = whisper.load_model("base")
result = model.transcribe("audio.mp3")

subs = pysubs2.load_from_whisper(result)
subs.save("audio.ass")
```

I'm not familiar with the Whisper library, so please correct me if necessary. However, from the `write_srt()` code it looks that it should be very easy to get the data.

**Comment by ArabCoders** [comment]
Indeed something similar yeah, that support pysubs2 wide formats

**Comment by tkarabela** [comment]
I've released version 1.5.0 of the library where `pysubs2.load_from_whisper(result)` as in the example above is implemented. Please upgrade :)

**Comment by ArabCoders** [comment]
Thank you for your quick implementation.
